### ISSUE TITLE
CMake project with Visual Studio 2019 Community compilation error.

### ISSUE THREAD

**Issue by 67-al-ahad** [MonaSolutions/MonaServer2#39]
Generated project are for MSVC++ 2019 and not 2015

Compiling MonaBase goes well.
Compiling MonaCore errors : 
```
onaCore\include\Mona/RTPWriter.h(116,1): message : Constructor for struct 'Mona::Media::Video::Tag' is declared 'explicit'
MonaCore\include\Mona/RTP_MPEG.h(38,7): message : see declaration of 'Mona::RTP_MPEG::writeVideo'
MonaCore\include\Mona/RTPWriter.h(114,1): error C2664: 'bool Mona::RTP_MPEG::writeAudio(Mona::Media::Audio::Tag,Mona::BinaryReader &,Mona::BinaryWriter &,Mona::UInt16)': cannot convert argument 1 from 'const TagType' to 'Mona::Media::Audio::Tag'
1>        with
1>        [
1>            TagType=Mona::Media::Audio::Tag
1>        ]

RTPWriter.h(116,1): error C2664: 'bool Mona::RTP_MPEG::writeVideo(Mona::Media::Video::Tag,Mona::BinaryReader &,Mona::BinaryWriter &,Mona::UInt16)': cannot convert argument 1 from 'const TagType' to 'Mona::Media::Video::Tag'
1>        with
1>        [
1>            TagType=Mona::Media::Video::Tag
1>        ]
```

**Comment by 67-al-ahad** [comment]
I removed all the RTP related files from source code and the code is compiling.
Can you fix this bug so that I can add them back again?

**Comment by 67-al-ahad** [comment]
Working version without RTP files.

https://github.com/67-al-ahad/MonaServer2

**Comment by demiantres** [comment]
Same problem here. It would be useful to have this fixed in the master branch.

**Comment by demiantres** [comment]
The problem has been fixed with path 7f59ad7.
